src/php/ext/grpc/byte_buffer.c

```diff
@@ -50,3 +52,31 @@ void byte_buffer_to_string(grpc_byte_buffer *buffer, char **out_string,
   *out_string = string;
   *out_length = length;
 }
+
+#else
+
+zend_string* byte_buffer_to_zend_string(grpc_byte_buffer *buffer) {
+  grpc_byte_buffer_reader reader;
+  if (buffer == NULL || !grpc_byte_buffer_reader_init(&reader, buffer)) {
+    /* TODO(dgq): distinguish between the error cases. */
```

Comment:
Probably no need to carry this comment forward

Reply:
of course, I'll delete it.